torch/_meta_registrations.py

```diff
@@ -1461,7 +1518,7 @@ def maybe_wrap_dim(dim: int, dim_post_expr: int, wrap_scalar: bool = True):
         dim_post_expr = 1
     min = -dim_post_expr
     max = dim_post_expr - 1
-    assert not (dim < min or dim > max)
+    assert not (dim < min or dim > max), f"dim {dim} out of bounds ({min}, {max})"
```

Comment:
should we leave a TODO here to switch to canonicalize_dim from primTorch instead of maybe_wrap_dim? `canonicalize_dim` properly throws index error.

Reply:
sure can do. One thing I will say is that it's nice having the names line up exactly with C++, makes porting easier

Reply:
Should we then use `maybe_wrap_dim` instead of `canonicalize` in primtorch?

Reply:
The c++ api isn't designed intentionally, it just exists. I would be happy to port over to an intentional api if there is one lol. But I am also trying to minimize work lol

Reply:
Hence a TODO, to minimize current work :-)